packages/jellyfish-api-core/src/category/governance.ts

```diff
@@ -0,0 +1,60 @@
+import BigNumber from 'bignumber.js'
+import { ApiClient } from '../.'
+
+export enum ProposalType {
+  COMMUNITY_FUND_REQUEST = 'CommunityFundRequest',
+  BLOCK_REWARD_RELLOCATION = 'BlockRewardRellocation',
+  VOTE_OF_CONFIDENCE = 'VoteOfConfidence'
+}
+
+export enum ProposalStatus {
+  VOTING = 'Voting',
+  REJECTED = 'Rejected',
+  COMPLETED = 'Completed'
+}
+
+/**
+ * Governance RPCs for DeFi Blockchain
+ */
+export class Governance {
+  private readonly client: ApiClient
+
+  constructor (client: ApiClient) {
+    this.client = client
+  }
+
+  /**
+   * Creates a Cummunity Fund Request
```

Comment:
can add a full stop here. same as the docs
```suggestion
   * Creates a Cummunity Fund Request.
```